robotoff/models.py

```diff
@@ -72,7 +72,7 @@ class ProductInsight(BaseModel):
     """Table to store the insights generated by Robotoff."""
 
     # ID is the unique ID for this insight.
-    id = peewee.UUIDField(primary_key=True)
+    id = peewee.UUIDField(primary_key=True, default=uuid.uuid4)
```

Comment:
What is this doing ?

Reply:
Setting a default value to UUID id when creating a ProductInsight